public/components/ctas/8.html

```diff
@@ -0,0 +1,26 @@
+<aside class="relative overflow-hidden text-gray-300 bg-gray-900 lg:flex justify-center">
```

Comment:
Classes that are not needed:

- `relative`
- `overflow-hidden`
- `lg:flex`
- `justify-center`

Reply:
Oops, this hasn't been addressed @worawutbat